app/2.2.x/kong-user.md

```diff
@@ -43,3 +43,9 @@ configuration property.
     ```sh
     kong start
     ```
+
+If running Kong as a non-root user other than the built-in `kong` user and group is desired, consider adding the user to the “kong” group with something like:
```

Comment:
```suggestion
If running Kong as a custom non-root user is preferred, add the user to the `kong` group with a command similar to the following (the command may differ depending on your OS):
```
I might be wrong, but it seems like you wouldn't want to say "something other than the built-in `kong` user and group" if you're still telling them to use the `kong` group.

Reply:
After looking at this flow again - I suggest moving this up as an optional step 2, because they'd want to do this before starting Kong. Eg:

2. (Optional) If running Kong ...etc

So then the task becomes:
1. Switch to user
2. Optionally assign a group
3. Start Kong